### DISCUSSION TITLE
Has there been any advancements with Ragged Arrays?

### DISCUSSION THREAD

**Discussion by bsaoptima** [jax-ml/jax#21975] [Q&A]
Hi there,

I'm trying to build some code that would require to `vmap` over a list of arrays which I am aware is not permitted and I looked through discussions and it has been mentioned that the team has been working on ragged arrays. 

Has there been any progress on this?

Thank you for your work!

**Comment by jakevdp** [comment]
In terms of public APIs, no, there's not really been any progress. @mattjj or @dougalm may be able to say more about design progress.
